--- a/src/tests/test_growth.f90	2026-09-02 06:21:05
+++ b/src/tests/test_growth.f90	2026-09-02 06:21:05
@@ -183,7 +183,7 @@
  dz      = 2.*sqrt(6.)/nx
  call set_boundary(-0.5,0.5,-0.25,0.25,-dz,dz)
  hfact   = hfact_default
- rhozero = 1.e-11/unit_density
+ rhozero = 1.0000000000000005e-11/unit_density
  totmass = rhozero*dxbound*dybound*dzbound
  if (onefluid) then
     rhog = rhozero * (1-dtgratio)
