--- a/src/tests/test_ptmass.f90
+++ b/src/tests/test_ptmass.f90
@@ -957,7 +957,7 @@
        npartoftype(igas) = 2
        npart        = 2
        xyzh(1:3,1:2)  = 5.*pos_fac
-       xyzh(4,1:2)    = 0.01
+       xyzh(4,1:2)    = 0.010000000000000004
        vxyzu(1:3,1) = [40.*vel_fac,40.*vel_fac,-10.*vel_fac]
        vxyzu(1:3,2) = [120.*vel_fac,120.*vel_fac,-30.*vel_fac]
        fxyzu(1:3,1:2) = 20.*acc_fac
